diff --git a/_lib/vite/vite.config.js b/_lib/vite/vite.config.js
index 7b33615eaf41ffaaedb25b6ad755edb7672cadd9..22a208dbcad40858fda0378929ad0fa79a2144cd 100644
--- a/_lib/vite/vite.config.js
+++ b/_lib/vite/vite.config.js
@@ -22,6 +22,23 @@ export default defineConfig(async () => {
             ...viteConfig?.build,
             cssCodeSplit: false,
         },
+        resolve: {
+          extensions: [
+            '.web.tsx',
+            '.web.ts',
+            '.web.js',
+            '.mjs',
+            '.mts',
+            '.ts',
+            '.tsx',
+            '.js',
+            '.jsx',
+            '.json',
+          ],
+          alias: {
+            'react-native': 'react-native-web',
+          },
+        },
         optimizeDeps: {
             ...(viteConfig.optimizeDeps ?? {}),
             include: [
